UnitConverter/ViewController.swift

```diff
@@ -17,10 +17,12 @@ class ViewController: UIViewController, UIPickerViewDelegate {
     
     private let converter = UnitConverter()
     
+    // MARK: View Lifecycle
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         let row = initialPickerRow()
-        celsiusPicker.selectRow(row, inComponent: 0, animated: false)
+        celsiusPicker.selectRow(row, inComponent: 0, animated: true)
```

Comment:
Leaving this to `false` is better here, because there is nothing to trigger the selection, and the animation likely isn't visible during `viewDidLoad`. Later, when students want to add another picker for F, to convert back and forth between F and C, we can identify a learning opportunity to show what the change to `true` does.